### PR TITLE
[NUI][AT-SPI] Static class Accessibility

### PR DESCRIPTION
**Pull request by aswigon**
### Description of Change ###

This class serves as a collection of unrelated methods, which are all static in DALi anyway, and as such is a good candidate for migration to a C# `static class`.

Complementary dali-csharp-binder patch (merge at the same time):
* https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/271723

### API Changes ###

 - ACR: None

### CODE REVIEW

**Comment by aswigon** [comment]
As I mentioned under https://github.com/Samsung/TizenFX/pull/3984, this class is a collection of methods from `Accessibility::Bridge` and `Accessibility::Accessible` which are all static, as well as functions from namespace `Dali::AtspiAccessibility` which do not belong to any class. That's why a `static class` is better than a singleton which does not correspond to an instance of any particular object in DALi.

**Review by lol-github — APPROVED** [review]
LGTM,  the code looks cleanlier now due to the reduction of number of parameters and due to the simplification of API usage: Accessibility.Accessibility.Instance.Say() ---> Accessibility.Accessibility.Say()

Optimally would be to have in public API as short form as Accessibility.Say() but as Artur noticed this is blocked by naming conflict with the namespace containing AccessibilityManager class.

File: src/Tizen.NUI/src/public/Accessibility/Accessibility.cs [hunk 1]
```diff
@@ -223,21 +179,21 @@ public void SetHighlightFrameView(View view)
...
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public View GetCurrentlyHighlightedView()
+        public static View GetCurrentlyHighlightedView()
         {
             var ptr = Interop.ControlDevel.DaliAccessibilityAccessibleGetCurrentlyHighlightedActor();
 
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
 
-            return this.GetInstanceSafely<View>(ptr);
+            return dummyHandle.GetInstanceSafely<View>(ptr);
```
Comment (kimcinoo) [inline src/Tizen.NUI/src/public/Accessibility/Accessibility.cs:188]: There is a way without 'this'. Cool.

**Review by kimcinoo — APPROVED** [review]
Looks very good to me!

**Comment by aswigon** [comment]
> Looks good to me except one thing. Could you remove this warning?
> 
> src/Tizen.NUI/src/public/Accessibility/Accessibility.cs#L36 Initialize all static fields in 'Accessibility' when those fields are declared and remove the explicit static constructor

Sure.

**Review by Seoyeon2Kim — APPROVED** [review]

**Pull request merged**